src/app/components/styles/app.scss

```diff
@@ -23,6 +23,27 @@
   }
 }
 
+/* contextual help */
+/* global style now to avoid layout shift when loading logic later */
+[data-article-id] {
+  &::after {
+    content: 'i';
+    cursor: wait;
```

Comment:
I wasn't able to see this cursor (nor the layout shift)

Reply:
The layout shift would have appeared without this change, only on first load of the page (the time it takes for the contextual help logic to load). It would push the text in the info list to the left when the "i" appears.
To see the opacity: 0.5 / cursor: wait in action, you'd probably need to simulate 3g network in the devtools and reload an entry page. It's still quite quick to load though in machines like ours.